netlib/network.go

```diff
@@ -0,0 +1,288 @@
+package netlib
```

Comment:
Can we please move this file to utils/netlib directory instead of creating top level netlib directory?